modules/dmrpp_module/DmrppArray.cc

```diff
@@ -2272,6 +2272,24 @@ bool DmrppArray::read()
                 }
                 break;
             }
+            case dods_float32_c: {
```

Comment:
One thing I've been thinking about is making it easier to write unit tests for our code. For this section, if these two case statement blocks called a functions to swap the bytes, then those functions would be easier to test in isolation. That might be one side effect of following the sonar scan advice to reduce the complexity.

The first block could be something like:

case dods_float32_c:
    swap_float32(get_buf());
    break;

Reply:
Make the swap separate functions.